pluto/src/color/core/color.spec.ts

```diff
@@ -133,4 +133,21 @@ describe("color.Color", () => {
       expect(c.pickByContrast(c1, c2)).toEqual(c1);
     });
   });
+  describe("grayness", () => {
+    const tests: Array<[string, number]> = [
+      ["#000000", 0],
+      ["#ffffff", 1],
```

Comment:
Does not make any sense to me why black is considered complete grayness and white is considered no grayness.